diff --git a/ogram Files (x86)/Microsoft Visual Studio/2019/Preview/MSBuild/Xamarin/Android/Xamarin.Android.Common - Copy.targets b/ogram Files (x86)/Microsoft Visual Studio/2019/Preview/MSBuild/Xamarin/Android/Xamarin.Android.Common.targets
index 271b611..48e58a6 100644
--- a/ogram Files (x86)/Microsoft Visual Studio/2019/Preview/MSBuild/Xamarin/Android/Xamarin.Android.Common - Copy.targets	
+++ b/ogram Files (x86)/Microsoft Visual Studio/2019/Preview/MSBuild/Xamarin/Android/Xamarin.Android.Common.targets	
@@ -475,7 +475,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
   <GetAdditionalResourcesFromAssemblies
       AndroidSdkDirectory="$(_AndroidSdkDirectory)"
       AndroidNdkDirectory="$(_AndroidNdkDirectory)"
-      Assemblies="@(ReferencePath);@(ReferenceDependencyPaths)"
+      Assemblies="@(ReferencePath_NoAndroidSupport);@(ReferenceDependencyPaths_NoAndroidSupport)"
       CacheFile="$(_AndroidResourcePathsCache)"
       YieldDuringToolExecution="$(YieldDuringToolExecution)"
       DesignTimeBuild="$(DesignTimeBuild)"
@@ -1269,7 +1269,7 @@ because xbuild doesn't support framework reference assemblies.
 		ResourceDirectory="$(MonoAndroidResourcePrefix)"
 		AdditionalResourceDirectories="@(LibraryResourceDirectories);@(_AdditionalAndroidResourcePaths->'%(Identity)\res')"
 		IsApplication="$(AndroidApplication)"
-		References="@(ReferencePath)"
+		References="@(ReferencePath_NoAndroidSupport)"
 		UseManagedResourceGenerator="True"
 		DesignTimeBuild="$(DesignTimeBuild)"
 		Condition="Exists ('$(MonoAndroidResourcePrefix)')"
@@ -1348,7 +1348,7 @@ because xbuild doesn't support framework reference assemblies.
 		ContinueOnError="$(DesignTimeBuild)"
 		CacheFile="$(_AndroidLibraryProjectImportsCache)"
 		DesignTimeBuild="$(DesignTimeBuild)"
-		Assemblies="@(ReferencePath);@(ReferenceDependencyPaths)"
+		Assemblies="@(ReferencePath_NoAndroidSupport);@(ReferenceDependencyPaths_NoAndroidSupport)"
 		AarLibraries="@(AndroidAarLibrary)"
 		ImportsDirectory="$(_LibraryProjectImportsDirectoryName)"
 		NativeImportsDirectory="$(_NativeLibraryImportsDirectoryName)"
@@ -1825,7 +1825,7 @@ because xbuild doesn't support framework reference assemblies.
 		ResourceDirectory="$(MonoAndroidResDirIntermediate)"
 		AdditionalResourceDirectories="@(LibraryResourceDirectories)"
 		IsApplication="$(AndroidApplication)"
-		References="@(ReferencePath)"
+		References="@(ReferencePath_NoAndroidSupport)"
 		UseManagedResourceGenerator="$(_UseManagedResourceGenerator)"
 		DesignTimeBuild="$(DesignTimeBuild)"
 	/>
@@ -1889,7 +1889,7 @@ because xbuild doesn't support framework reference assemblies.
 <Target Name="_RunManagedAidlTool">
   <GenerateManagedAidlProxies
     SourceAidlFiles="@(AndroidInterfaceDescription)"
-    References="@(ReferencePath);@(ReferenceDependencyPaths)"
+    References="@(ReferencePath_NoAndroidSupport);@(ReferenceDependencyPaths_NoAndroidSupport)"
     IntermediateOutputDirectory="$(MonoAndroidIntermediate)"
     ParcelableHandlingOption="$(ManagedAidlParcelableHandling)"
     OutputNamespace="$(ManagedAidlNamespace)" />
@@ -2298,8 +2298,8 @@ because xbuild doesn't support framework reference assemblies.
   Inputs="$(MSBuildAllProjects);@(_ResolvedAssemblies);$(_AndroidManifestAbs);$(_AndroidBuildPropertiesCache);@(_AndroidResourceDest)"
   Outputs="$(IntermediateOutputPath)_javastubs.stamp">
   <GenerateJavaStubs
-    ResolvedAssemblies="@(_ResolvedAssemblies)"
-    ResolvedUserAssemblies="@(_ResolvedUserAssemblies)"
+    ResolvedAssemblies="@(_ResolvedAssemblies_NoAndroidSupport)"
+    ResolvedUserAssemblies="@(_ResolvedUserAssemblies_NoAndroidSupport)"
     ErrorOnCustomJavaObject="$(AndroidErrorOnCustomJavaObject)"
 	ManifestTemplate="$(_AndroidManifestAbs)"
 	MergedManifestDocuments="@(ExtractedManifestDocuments)"
@@ -2350,8 +2350,8 @@ because xbuild doesn't support framework reference assemblies.
   Outputs="$(_AndroidStampDirectory)_GeneratePackageManagerJava.stamp">
   <!-- Create java needed for Mono runtime -->
   <GeneratePackageManagerJava
-    ResolvedAssemblies="@(_ResolvedAssemblies)"
-    ResolvedUserAssemblies="@(_ResolvedUserAssemblies)"
+    ResolvedAssemblies="@(_ResolvedAssemblies_NoAndroidSupport)"
+    ResolvedUserAssemblies="@(_ResolvedUserAssemblies_NoAndroidSupport)"
     MainAssembly="$(MonoAndroidLinkerInputDir)$(TargetFileName)"
     OutputDirectory="$(IntermediateOutputPath)android\src\mono"
     EnvironmentOutputDirectory="$(IntermediateOutputPath)android\src\mono\android\app"
@@ -2833,7 +2833,7 @@ because xbuild doesn't support framework reference assemblies.
 	SupportedAbis="$(_BuildTargetAbis)"
 	AndroidSequencePointsMode="$(_SequencePointsMode)"
 	AotAdditionalArguments="$(AndroidAotAdditionalArguments)"
-	ResolvedAssemblies="@(_ResolvedUserAssemblies);@(_ShrunkFrameworkAssemblies)"
+	ResolvedAssemblies="@(_ResolvedUserAssemblies_NoAndroidSupport);@(_ShrunkFrameworkAssemblies)"
 	AotOutputDirectory="$(_AndroidAotBinDirectory)"
 	IntermediateAssemblyDir="$(MonoAndroidIntermediateAssemblyDir)"
 	LinkMode="$(AndroidLinkMode)"
@@ -2848,7 +2848,7 @@ because xbuild doesn't support framework reference assemblies.
 	Condition=" '$(AndroidAotMode)' == 'Hybrid' And '$(AotAssemblies)' == 'True' "
 	AndroidAotMode="$(AndroidAotMode)"
 	ToolPath="$(_MonoAndroidToolsDirectory)"
-	ResolvedAssemblies="@(_ResolvedAssemblies)">
+	ResolvedAssemblies="@(_ResolvedAssemblies_NoAndroidSupport)">
   </CilStrip> 
 
   <!-- Bundle the assemblies into native libraries in the apk -->
@@ -2856,7 +2856,7 @@ because xbuild doesn't support framework reference assemblies.
 		Condition="'$(BundleAssemblies)' == 'True'"
 		KeepTemp="$(AndroidMakeBundleKeepTemporaryFiles)"
 		AndroidNdkDirectory="$(_AndroidNdkDirectory)"
-		Assemblies="@(_ResolvedUserAssemblies);@(_AndroidResolvedSatellitePaths);@(_ShrunkFrameworkAssemblies)"
+		Assemblies="@(_ResolvedUserAssemblies_NoAndroidSupport);@(_AndroidResolvedSatellitePaths);@(_ShrunkFrameworkAssemblies)"
 		IncludePath="$(MonoAndroidIncludeDirectory)"
 		SupportedAbis="$(_BuildTargetAbis)"
 		TempOutputPath="$(IntermediateOutputPath)"
@@ -2872,11 +2872,11 @@ because xbuild doesn't support framework reference assemblies.
     BundleAssemblies="$(BundleAssemblies)"
     BundleNativeLibraries="$(_BundleResultNativeLibraries)"
     EmbedAssemblies="$(EmbedAssembliesIntoApk)"
-    ResolvedUserAssemblies="@(_ResolvedUserAssemblies);@(_AndroidResolvedSatellitePaths)"
+    ResolvedUserAssemblies="@(_ResolvedUserAssemblies_NoAndroidSupport);@(_AndroidResolvedSatellitePaths)"
     ResolvedFrameworkAssemblies="@(_ShrunkFrameworkAssemblies)"
     NativeLibraries="@(AndroidNativeLibrary)"
     AdditionalNativeLibraryReferences="@(_AdditionalNativeLibraryReferences)"
-    EmbeddedNativeLibraryAssemblies="$(OutDir)$(TargetFileName);@(ReferencePath);@(ReferenceDependencyPaths)"
+    EmbeddedNativeLibraryAssemblies="$(OutDir)$(TargetFileName);@(ReferencePath_NoAndroidSupport);@(ReferenceDependencyPaths_NoAndroidSupport)"
     DalvikClasses="@(_DexFile)"
     SupportedAbis="$(_BuildTargetAbis)"
     CreatePackagePerAbi="$(AndroidCreatePackagePerAbi)"
